packages/jsdoc-plugins/lib/statics-augmentor/statics-augmentor.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Frederico Knabben. All rights reserved.
+ * Licensed under the terms of the MIT License (see LICENSE.md).
+ */
+
+'use strict';
+
+/**
+ * Manually adds doclets for inherited static members.
+ * Jsdoc doesn't support inheritance of static members which is why this plugin is needed.
+ */
+class StaticsAugmentor {
+	/**
+	 * @param {Array.<Doclet>} doclets
+	 */
+	constructor( doclets ) {
+		this._data = doclets;
+	}
+
+	/**
+	 * Main method responsible for inheriting static members.
+	 */
+	augmentStatics() {
```

Comment:
Can we export this method only? The class here is not necessary – it maintains no state.